packages/framework/composition/README.md

```diff
@@ -1 +1,47 @@
-# @fluentui-react-native/use-styling
+# @fluentui-react-native/composition
+
+This package contains the `composeFactory` which all components created using the `compose` function under `@fluentui-react-native/framework` use to construct the component itself. The `composeFactory` returns a [staged component](https://github.com/microsoft/fluentui-react-native/tree/master/packages/framework/use-slot) with two additional functions, `customize` and `compose`. These functions allow for these components to have a degree of extensibility.
```

Comment:
For the first words that people read in relation to compose, I feel there'a bit too much inner technical deatil (`composeFactory` / `stagedComponent`). I think it might be better if the first few sentences were a rougher overview. Something like "Compose is a way to make Higher Order Components that expose some methods (customize  / compose) to further style your components". Not that exactly.. but maybe save talking about `composeFactory` / `stagedComponent` for a "Behind the Scenes" section.

Happy to take alternative views though, and feel free to shoot the suggestion down!

Reply:
Good point, I can reword the first parts. Thanks for the feedback!

Reply:
Tried rewording, let me know if this is better